pkg/util/grpcclient/report.go

```diff
@@ -0,0 +1,45 @@
+package grpcclient
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"google.golang.org/grpc"
+
+	dmiapi "github.com/kubeedge/mappers-go/pkg/apis/dmi/v1"
+)
+
+// ReportDeviceStatus report device status to edgecore
+func ReportDeviceStatus(request *dmiapi.ReportDeviceStatusRequest) error {
+	conn, err := grpc.Dial(cfg.Common.EdgeCoreSock,
+		grpc.WithInsecure(),
+		grpc.WithBlock(),
+		grpc.WithContextDialer(
+			func(ctx context.Context, s string) (net.Conn, error) {
+				unixAddress, err := net.ResolveUnixAddr("unix", cfg.Common.EdgeCoreSock)
+				if err != nil {
+					return nil, err
+				}
+				return net.DialUnix("unix", nil, unixAddress)
+			},
+		),
+	)
+	if err != nil {
+		return fmt.Errorf("did not connect: %v", err)
+	}
+	// 延迟关闭连接
```

Comment:
the Chinese comments in this file can be all removed.

Reply:
done